streaming-pacman/pacman/game/js/pacman.js

```diff
@@ -267,10 +269,29 @@ function killingPacman() {
 		clearInterval(PACMAN_KILLING_TIMER);
 		PACMAN_KILLING_TIMER = -1;
 		erasePacman();
-		if (LIFES > 0) { 
+		if (LIFES > 0) { //TODO: need to add lifes = -1 event to user_game to define end of game
 			lifes(-1);
 			setTimeout('retry()', (PACMAN_RETRY_SPEED));
-		} else { 
+		}
+		else if (LIFES === 0) {
+
+			var record = {};
+			record.user = window.name;
+			record.score = SCORE;
+
+			if (confirm('Watch ad to continue?')) {
```

Comment:
can we control this behavior from a variable? To enable/disable it?
The confirm function might be blocked by some adblock/phones etc...
For now would be enough to add a variable in the pacman.js file itself, at the very top and document it in the readme (so people can disable it if they need to)